common/notification/actions.ts

```diff
@@ -551,12 +551,20 @@ const _notificationActions = {
         },
     },
     student_add_appointment_match: {
-        description: 'Tutee / Tutor added Match Appointment',
+        description: 'Tutee / Tutor added Match Appointment (Pupil event)',
         sampleContext: {
             student: sampleUser,
             matchId: '1',
         },
     },
+    student_add_appointment_match_with_pupil: {
+        description: 'Tutee / Tutor added Match Appointment (Tutee / Tutor event)',
```

Comment:
It's the same as the action before. The difference is that `student_add_appointment_match` is called for the pupil and `student_add_appointment_match_with_pupil` for the student.

Reply:
Urgh, naming conventions ... initial plan was to prefix the action id with the user for which the action is triggered, not the user that triggers the action for "student_add_appointment_match" that obviously went wrong ...

Reply:
Yeah 😞 
Unfortunately, it's not easy to enforce these kind of conventions and it's even harder to change the naming afterwards.